tidb-server/main.go

```diff
@@ -492,6 +493,8 @@ func setGlobalVars() {
 
 	tikv.CommitMaxBackoff = int(parseDuration(cfg.TiKVClient.CommitTimeout).Seconds() * 1000)
 	tikv.PessimisticLockTTL = uint64(parseDuration(cfg.PessimisticTxn.TTL).Seconds() * 1000)
+
+	autoid.SetStep(cfg.Performance.AutoIDStep)
```

Comment:
Please remove the comment:

https://github.com/pingcap/tidb/blob/cf5f42b8ece0fad2b3f93c6753747b392f33f4c8/meta/autoid/autoid.go#L70